index.js

```diff
@@ -335,7 +335,7 @@ $inherit(HTMLElement, Node, {
    * Get first child node
    * @return {Node} first child node
    */
-  get firstChild() {
+   get firstChild() {
```

Comment:
```suggestion
  get firstChild() {
```

Reply:
Keep indents with other function defs in the project.

Reply:
@MakotoNakai The later one is resolved, but not this one. It still has an extra space before `get`